Stytch/DemoApps/StytchB2BUIDemo/Info.plist

```diff
@@ -9,7 +9,7 @@
 			<string>Editor</string>
 			<key>CFBundleURLSchemes</key>
 			<array>
-				<string>stytch-demo</string>
+				<string>stytchui-public-token-test-b6be6a68-d178-4a2d-ac98-9579020905bf</string>
 			</array>
```

Comment:
I want to leave this in place for the duration of this feature branch, what do you think?